tests/ut/ir/parser/test_window_op.py

```
@@ -154,7 +154,7 @@ def host_orch(self, x: pl.Tensor[[64], pl.FP32]):
 
 
 def test_window_rejects_unknown_kwarg():
-    with pytest.raises(Exception, match="does not accept kwarg"):
+    with pytest.raises(Exception, match="unexpected keyword argument|does not accept kwarg"):
```

Ruff (0.15.12)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)